bbq/vm/test/interpreter_test.go

```diff
@@ -641,8 +641,6 @@ func BenchmarkRuntimeFungibleTokenTransfer(b *testing.B) {
 		},
 	}
 
-	environment := runtime.NewBaseInterpreterEnvironment(runtime.Config{})
-
```

Comment:
Avoid passing the environment explicitly., The test util will create the correct environment based on the `compile` flag.

Reply:
ah I see. Added the envrionement back: https://github.com/onflow/cadence/pull/4075/commits/8d3357059bd8be060a6127a996a1aa1b7fe97edd